tests/system/test_aws_s3.py

```diff
@@ -108,3 +114,41 @@ def test_ingest_single_parquet_file(self, use_datastore_profile):
             name="overwrite-pq-spec-path", entities=[fstore.Entity("name")]
         )
         fstore.ingest(fset, df1, targets=targets)
+
+    def test_ingest_with_parquet_source(self, use_datastore_profile):
+        #  create source
+        s3_fs = fsspec.filesystem(
+            "s3", key=self._access_key_id, secret=self._secret_access_key
+        )
+        param = self.s3["ds"] if use_datastore_profile else self.s3["s3"]
+        data = {"Column1": [1, 2, 3], "Column2": ["A", "B", "C"]}
+        df = pd.DataFrame(data)
+        source_path = (
+            f"{os.path.dirname(param['parquet_url'])}/source_{uuid.uuid4()}.parquet"
+        )
+        with tempfile.NamedTemporaryFile(mode="w+", delete=True) as temp_file:
+            df.to_parquet(temp_file.name)
+            path_only = source_path.replace("ds://s3ds_profile/", "").replace(
+                "s3://", ""
+            )
+            s3_fs.put_file(temp_file.name, path_only)
+        parquet_source = ParquetSource(name="test", path=source_path)
+
+        # ingest
+        targets = [
+            ParquetTarget(
+                path=f"{os.path.dirname(param['parquet_url'])}/target_{uuid.uuid4()}"
+            )
+        ]
+        fset = fstore.FeatureSet(
+            name="test_fs",
+            entities=[fstore.Entity("Column1")],
+        )
+
+        fstore.ingest(fset, source=parquet_source, targets=targets)
+        result = targets[0].as_df()
```

Comment:
I discovered that we are no longer support as_df, so please use `fset.get_target_path()`.
In this way you can get the result with the previous change:
```suggestion
        result = ParquetSource(path=fset.get_target_path()).to_dataframe()
```